flower/views/broker.py

```diff
@@ -24,8 +24,12 @@ def get(self):
             http_api = app.options.broker_api
 
         broker = Broker(app.capp.connection().as_uri(include_password=True),
-                        http_api=http_api)
+                        http_api=http_api, celery_conf=self.capp.conf)
         queue_names = ControlHandler.get_active_queue_names()
```

Comment:
there are so many other options that could be used, so idea was to pass all of them once, to have them ready if some other options will be needed (e.g. for mongodb broker, those options are under different name CELERY_MONGODB_BACKEND_SETTINGS)